### DISCUSSION TITLE
pseudo-element not displaying correctly

### DISCUSSION THREAD

**Discussion by 41an** [AnubisNekhet/AnuPpuccin#219] [Snippets]
bro would ur theme cause any issues with the CSS :after pseudo-element not displaying correctly？

**Comment by syrth-io** [comment]
it's been a few months so I'm not sure if you're still having problems, but it's unfortunately impossible to give a definitive answer to this without seeing your code or knowing whether you have any other snippets or plugins that might be overriding the specificity, or if it actually works in the manner you're attempting. because Obsidian has so much complex CSS to begin with that changes with updates sometimes, and any appearance-changing add-on further complicates it, it could even be multiple things. if you still need help, can you give me a bit more detail and show me the CSS you're trying to achieve?
